recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_git.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "The RAUC hawkBit updater operates as an interface between the RAUC D-Bus API and the hawkBit DDI API."
+HOMEPAGE = "https://github.com/rauc/rauc-hawkbit-updater"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1a6d268fd218675ffea8be556788b780"
+
+SRC_URI = "git://github.com/rauc/rauc-hawkbit-updater.git;protocol=https"
+SRCREV = "d909982e9e4b84cb76b98bf85f25a0a88472301a"
```

Comment:
This commit id is from april 2020. can we use a newer one?

Reply:
Is there a specific reason to update to a newer version? Anything that does not work with this revision?

Reply:
yes i get a Segmentation fault with this version. The commit cc2cd6e2f55c547d78a365f024a2ab5e40bbb22e from 13th April 2021 runs and have more details in the debug messages.
Additional there are fixed bugs, more tests and many other changes.

Reply:
Ok, seems reasonable. I think @ctenruh-phytec should update the version in meta-yogurt instead of in this layer. We must be careful though, with choosing the correct version. We must avoid maintaining a newer version in zeus than meta-rauc does in dunfell and later. Revision `28900b59859d80d2e5275ad03278b0b76e7f1b51` seems to be OK, but please double check this. See also https://github.com/rauc/meta-rauc/blob/dunfell/recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_git.bb.